subprojects/common/time/src/testFixtures/kotlin/com/avito/time/TimeMachineProvider.kt

```diff
@@ -10,25 +9,25 @@ import java.util.concurrent.TimeUnit
  */
 public class TimeMachineProvider : TimeProvider {
 
-    private var timeShift: Long = 0
+    public var now: Long = 0
 
     private val defaultTimeProvider = DefaultTimeProvider()
 
     public fun moveForwardOn(time: Long, unit: TimeUnit = TimeUnit.MILLISECONDS) {
-        timeShift += unit.toMillis(time)
+        now += unit.toMillis(time)
     }
 
     public fun moveBackOn(time: Long, unit: TimeUnit = TimeUnit.MILLISECONDS) {
-        timeShift -= unit.toMillis(time)
+        now -= unit.toMillis(time)
     }
 
-    override fun nowInMillis(): Long = defaultTimeProvider.nowInMillis() + timeShift
```

Comment:
It's a strange implementation because time will be random because of using defaultTimeProvider.
Now time machine will control the current time